### PR TITLE
`General`: Improve usability of LTI launch

### PR DESCRIPTION
**Pull request by manuelmanso**
#### General
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client/) and ensured that the layout is responsive.
- [x] Following the [theming guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client-design/), I specified colors only in the theming variable files and checked that the changes look consistent in both the light and the dark theme.
- [x] I added multiple integration tests (Jest) related to the features (with a high test coverage), while following the [test guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client-tests/).
- [x] I documented the TypeScript code using JSDoc style.
- [x] I added multiple screenshots/screencasts of my UI changes.

### Motivation and Context
Improve the QoL of an LTI launch by prefilling the username field with the corresponding username for the LTI launch. That way, users can more easily now what user they are logging in as and only need to fill in the password.

Also prevents user from using ESC to close the initializer modal showing the password, forcing them to actually read the content and save the password. Adds a copy button for the password as well

### Description

- Prefills the username field with the corresponding username from the LTI launch
- Prevent users from closing modal with ESC (keyboard: false)
- Add copy button for password field

### Steps for Testing
Prerequisites:
- 1 Instructor

Launch as new LTI student 
1. Verify that password copy button is present in the password modal and that it works
2. Verify that it's not possible to close modal with "ESC" key (or any other key)

Launch as existing LTI student 
1. Verify that notification is shown "LTI launch as {username}" with the correct username
2. Verify that the username is already prefilled in the login bar

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2
### Test Coverage

| Class/File | Branch | Line |
|------------|-------:|-----:|
| account.service.ts | 87% | 88% |
| user-route-access.service.ts | 78% | 89% |

### Screenshots

Password initializer modal with copy button

image

### CODE REVIEW

File: src/test/javascript/spec/service/account.service.spec.ts [hunk 1]
```diff
@@ -14,14 +14,17 @@ import { Course } from 'app/entities/course.model';
 import { Exercise } from 'app/entities/exercise.model';
 import { Participation } from 'app/entities/participation/participation.model';
 import { Team } from 'app/entities/team.model';
+import { post } from '../helpers/sample/metis-sample-data';
```
Comment (JohannesStoehr) [inline src/test/javascript/spec/service/account.service.spec.ts:17]: Unused import

**Review by JohannesStoehr — CHANGES_REQUESTED** [review]
Apart from this, code looks good

File: src/test/javascript/spec/service/account.service.spec.ts (hunk 1 above)
Comment (manuelmanso) [reply src/test/javascript/spec/service/account.service.spec.ts]: Thanks, fixed!

**Review by frederik-kt — APPROVED** [review]
Tested on TS1. Could not close modal and username was prefilled correctly.

**Review by rriyaldhi — APPROVED** [review]
Tested on TS1 and works according to the testing steps.

**Review by danielbode — APPROVED** [review]
Tested on ts1 during a testing session. Works as expected!

**Review by JohannesStoehr — APPROVED** [review]
Thanks for fixing the code issue. Code looks good to me now

**Pull request merged**